internal/clients/http/sources/sources_client.go

```diff
@@ -232,7 +236,11 @@ func (c *sourcesClient) loadAppId(ctx context.Context) (string, error) {
 		logger.Warn().Err(err).Msg("Failed to fetch ApplicationTypes from sources")
 		return "", fmt.Errorf("failed to fetch ApplicationTypes: %w", err)
```

Comment:
```suggestion
		return "", fmt.Errorf("Failed to fetch ApplicationTypes: %w", err)
```

Reply:
Errors should start with a small letter :)